### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by vishalg8454**
This first PR adds the first page to the app. The web app is responsive on both mobile and web.

The components added are-

- `Navbar` at src/components/Navbar.jsx
- `NavigationCollapsed` at src/components/navigation/navigation-collapsed/NavigationCollapsed.jsx
- `NavigationExpanded` at src/components/navigation/navigation-expanded/NavigationExpanded.jsx
- `NavigationBottom` at src/components/navigation/navigation-bottom/NavigationBottom.jsx

The routes added are

- `Homepage` at ("/") at src/pages/homepage/HomePage.jsx

The hooks added is

- `useMedia` at src/custom-hooks/useMedia.js

# [live preview](https://deploy-preview-1--videostock.netlify.app/)

image

image

Would appreciate you feedback and suggestions.

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export { Navbar } from "./navbar/Navbar";
```
Comment (amirnathani9) [inline src/components/index.js:1]: u can export this files by making the index file in navigation folder

**Review by amirnathani9 — APPROVED** [review]
Liked the use of useMedia Hook.
The Ui is Clean 
Code is Clean and easy to ready
File structure is good

One suggestion is try to keep one PR upto 500 line of code

File: src/components/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+export { NavigationCollapsed } from "./navigation/navigation-collapsed/NavigationCollapsed";
+export { NavigationExpanded } from "./navigation/navigation-expanded/NavigationExpanded";
+export { NavigationBottom } from "./navigation/navigation-bottom/NavigationBottom";
```
Comment (amirnathani9) [inline src/components/index.js:4]: u can export the components by making the index file in the navigation folder

**Review by surbhi84 — APPROVED** [review]
- The code looks neat.
- Great PR description.
- Liked that you've used Navlinks for sidebar.
- The index.css has commented code, remove if not needed.

**Pull request merged**